src/client/java/minicraft/core/Renderer.java

```diff
@@ -327,6 +328,15 @@ private static void renderGui() {
 			Font.drawBackground(dura + "%", screen, 164, Screen.h - 16, Color.get(1, 255 - green, green, 0));
 		}
 
+		// WATERING CAN CONTAINER STATUS
+		if (player.activeItem instanceof WateringCanItem) {
+			// Draws the text
+			WateringCanItem tin = (WateringCanItem) player.activeItem;
+			int dura = tin.content * 100 / tin.CAPACITY;
+			int green = (int)(dura * 2.55f); // Let duration show as normal.
+			Font.drawBackground(dura + "%", screen, 164, Screen.h - 16, Color.get(1, 255 - green, green, 0));
+		}
+
```

Comment:
What does this look like in game?

Reply:
https://github.com/MinicraftPlus/minicraft-plus-revived/assets/74168521/0502e24a-8aab-48e5-848d-77969460e412

Reply:
3600 ticks, 60 seconds.

Reply:
The watering can.

Reply:
Could you reduce the capacity of the watering can?

Reply:
Ok. I reduced a half of the capacity.

Reply:
Great!